lib/components/map/default-map.tsx

```diff
@@ -310,6 +310,7 @@ class DefaultMap extends Component {
           <EndpointsOverlay />
           <RouteViewerOverlay />
           <TransitVehicleOverlay ModeIcon={ModeIcon} />
+          <GeolocateControl position="bottom-left" />
```

Comment:
How about top-left? Currently, in the desktop view, the geolocate button position conflicts with the attribution (the attribution text appears on top, so the geolocate button cannot be activated through the mouse. In the mobile view, once the results are shown, the geolocate button is hidden by the Expand/Collapse Map button.

Reply:
I think the geolocate button is very useful.